libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Microsoft.AdaptiveDialog.uischema

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "https://schemas.botframework.com/schemas/ui/v1.0/ui.schema",
```

Comment:
We should update all of the  .csproj files to include a content rule to include .ui schema
     <content include="**/*.uischema"/ >